core/replay/replay.py

```diff
@@ -202,6 +202,21 @@ def main():
             aws_service_helper.s3_put_object(
                 f, bucket["bucket_name"], f"{bucket['prefix']}{object_key}"
             )
+        
+        logger.info(f"Uploading replay worker logs to {bucket['bucket_name']}/{bucket['prefix']}")
```

Comment:
Can this be combined with the method above? Maybe something like:
```
for log_type in [replay_logs, replay_worker_logs]:
        dir = f"core/logs/replay_log-{replay_id}"
        object_key = f"{log_type}.zip"
        zip_file_name = f"{log_type}.zip"
        with zipfile.ZipFile(zip_file_name, "w", zipfile.ZIP_DEFLATED) as zip_object:
            for folder_name, sub_folders, file_names in os.walk(dir):
                for filename in file_names:
                    file_path = os.path.join(folder_name, filename)
                    zip_object.write(file_path)
        with open(zip_file_name, "rb") as f:
            aws_service_helper.s3_put_object(
                f, bucket["bucket_name"], f"{bucket['prefix']}{object_key}"
            )
```

Reply:
I also had to add the directories as the dir are different for each of the files. Modified the code, please take a look.